modules/ROOT/pages/to-download-and-install-studio-wx.adoc

```diff
@@ -2,24 +2,62 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:page-aliases: jdk-requirement-lnx-worflow.adoc, jdk-requirement-wx-workflow.adoc, jdk-requirement-xos-worflow.adoc, studio-configure-vm-task-lnx.adoc, studio-configure-vm-task-unx.adoc, studio-configure-vm-task-wx.adoc, faq-jdk-requirement.adoc
 
-* Before downloading and launching Anypoint Studio, make sure you have the Java SE JDK 8 installed.
+== Prerequisites
 
-To install Anypoint Studio:
+* Ensure that you have the Java SE JDK 8 installed and configured.
+* Download the Anypoint Studio installation file from the https://www.mulesoft.com/lp/dl/studio[Anypoint Studio Download Site].
 
-. Download Anypoint Studio from the https://www.mulesoft.com/lp/dl/studio[Anypoint Studio Download Site] and extract it into the 'C:\' root folder. +
+[NOTE]
+Anypoint Studio 7.x only supports Mule 4.x projects since the structure of the project, export format, XML and scripting language are different.
+
+After downloading and installing the JDK, ensure that your JDK installation is properly configured within the the `JAVA_HOME` environment variable by following the instructions below.
+
+. In Search, search for `System (Control Panel)` and select it.
+. Click *Advanced System Settings*.
+. Click *Environment Variables*.
+. In *System Variables*, select the `PATH` environment variable.
+. Click *Edit*.
++
+If the `PATH` environment variable does not exist, click *New*.
+. In *Edit System Variable* (or *New System Variable*), ensure that the JDK 1.8.0 `/bin` directory is the first item in your `PATH` environment variable, and click *OK*. +
```

Comment:
See suggestion above and verify that regardless of editing PATH or creating New, this step is the same.